### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by allisonzheng**
adds estimate-area documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,6 +151,40 @@ tilesets delete-source
...
+```shell
+tilesets estimate-area <features> -p/--precision <precision>
+```
+
+Estimates area (km<sup>2</sup>) of features with a specified precision. Accepts line-delimited GeoJSON or GeoJSON feature collections as files or via `stdin` or a list of string-encoded coordinate pairs of the form "[lng, lat]", or "lng, lat", or "lng lat". Precision must be 10m, 1m, 30cm or 1cm.
+
+Flags:
+
+* `--no-validation` [optional]: do not validate source data locally before area calculation
+* `--force-1cm` [optional]: enables 1cm precision
```
Comment (mapsam) [inline README.md:165]: What does "forcing 1cm precision" mean?

File: README.md [hunk 2]
```diff
@@ -150,6 +151,40 @@ tilesets delete-source
...
 
+### estimate-area 
+
+```shell
+tilesets estimate-area <features> -p/--precision <precision>
+```
+
+Estimates area (km<sup>2</sup>) of features with a specified precision. Accepts line-delimited GeoJSON or GeoJSON feature collections as files or via `stdin` or a list of string-encoded coordinate pairs of the form "[lng, lat]", or "lng, lat", or "lng lat". Precision must be 10m, 1m, 30cm or 1cm.
+
+Flags:
```
Comment (mapsam) [inline README.md:162]: Should the `--precision` flag be documented in this list as well? I found it hard to find the possible options in the description above.

Additionally, it'd be nice to have a `[required]` note next to it if it is required, or note if there's a default (similar to the [create](https://github.com/mapbox/tilesets-cli#create) command)

File: README.md [hunk 3]
```diff
@@ -150,6 +151,40 @@ tilesets delete-source
 
 Permanently delete a tileset source and all of its files. This is not a recoverable action!
 
+### estimate-area 
+
+```shell
+tilesets estimate-area <features> -p/--precision <precision>
+```
+
+Estimates area (km<sup>2</sup>) of features with a specified precision. Accepts line-delimited GeoJSON or GeoJSON feature collections as files or via `stdin` or a list of string-encoded coordinate pairs of the form "[lng, lat]", or "lng, lat", or "lng lat". Precision must be 10m, 1m, 30cm or 1cm.
```
Comment (mapsam) [inline README.md:160]: Does this calculate _tiled_ area? If so, let's note this so users aren't surprised by small features and large areas

File: README.md [hunk 4]
```diff
@@ -150,6 +151,40 @@ tilesets delete-source
...
+
+* `--no-validation` [optional]: do not validate source data locally before area calculation
+* `--force-1cm` [optional]: enables 1cm precision 
+
+Usage
+
+```shell
+
+# GeoJSON feature files
+tilesets estimate-area ./file.geojson ./file.geojson -p <precision>
```
Comment (mapsam) [inline README.md:172]: is `./file.geojson` duplicated intentionally? If this can take multiple files, maybe name one of them `file1` and `file2` for clarity.

**Review by mapsam — CHANGES_REQUESTED** [review]
Thanks @allisonzheng! Left a few comments asking for some extra clarity.

File: README.md (hunk 1 above)
Comment (allisonzheng) [reply README.md:165]: it's a flag you have to put to do 1cm precision, it'll throw an error if you choose 1cm without it. Should I add it to the description? I didn't want to make it too wordy

Comment (mapsam) [reply README.md:165]: That's good to know! Brainstorming some ideas:

> enables 1cm zoom calculations. The 1cm (zoom >16) precision is not available in the Tilesets API by default. Reach out to support to enable this feature.
> link: https://support.mapbox.com/hc/en-us/requests/new?ticket_form_id=360000291231

Grabbed this from the docs at https://docs.mapbox.com/mapbox-tiling-service/reference/#zoom-levels

**Review by mapsam — APPROVED** [review]
LGTM! :shipit:
